appdev/webapp.rst

```diff
@@ -0,0 +1,55 @@
+Creating a Webapp
+=================
+
+Webapps are small applications, that load content in a browser. This tutorial for ubuntu will just create a starter for a webpage, that can be displayed easily in a mobile device.
+
+
+alternate-webapp-generator
+--------------------------
+
+The alternate-webapp-generator will download all the stuff you need for a webapp and will add the the url, icon and a lot of meta-information to the app.
+In the end a .click package will be created, which can be uploaded to the OpenStore
+
+Download the following zip-file and unzip
+::
+
+    https://github.com/Winael/ubuntu/blob/master/ubuntuphone/webapps/Reboot/alternate-webapp-generator.zip
+
+Make the app-creator executable:
+::
+
+    chmod +x /path/to/alternate-webapp-generator.sh
+
+Now we can start and edit the config.cfg.
```

Comment:
What is a config.cfg, something ducks walk on? Jk, but this comes a little out of the blue. Maybe explain above that in the process we'll create a new file that holds all the information, called config.cfg.

Reply:
Its included in the zip package - i will add a line with this information